pontoon/checks/libraries/compare_locales.py

```diff
@@ -45,7 +45,18 @@ def __init__(self, key, raw_val, pre_comment):
 
     @property
     def all(self):
-        return u'<!ENTITY {} \"{}\">'.format(self.key, self.raw_val)
+        # Wrap entities with apostrophes when they contain quotes
+        # Related bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1479208
+        if '"' in self.raw_val:
+            wrap = "'"
+        else:
+            wrap = '"'
```

Comment:
Thanks for addressing this bug so quickly!

This will wrap `Don't press \"Enter\".` like this: `'Don't press \"Enter\".'`. I checked and we don't have such strings in the DB, but we could, so I suggest we stay on the safe side.

Reply:
That's what pondered me too and that's why I decided to start by doing this PR instead of going too deep.